packages/ui/src/routes/send/SendPage.tsx

```diff
@@ -179,17 +194,25 @@ const SendPage = () => {
                         }}
                         debounced
                     />
+                    {displayAddToContacts && (
```

Comment:
I'd simply check whether the address is a contact or not here, rather than having a callback from `<AccountSearchResults ... />`.

Or we can have a second parameter in the  `onAccountSelect` cb.


```javascript
const onAccountSelect(account:AccountInfo, isContact:boolean) => {

}
```

Reply:
Done! 💪